pkgs/development/python-modules/jep/default.nix

```
@@ -0,0 +1,42 @@
+{ lib, python, fetchPypi, buildPythonPackage, numpy, jre, numpySupport ? true }:
```

```suggestion
{ lib, python, fetchPypi, buildPythonPackage, numpy, jre }:
```
or is there any compelling reason to turn it off?

---

Not really. Their build does different things depending on whether numpy is present or not, which is why I wanted to expose that option.